libbeat/_meta/config/setup.template.reference.yml.tmpl

```diff
@@ -33,6 +33,9 @@
 # Name under which the template is stored in Elasticsearch
 #setup.template.json.name: ""
 
+# Set this option if the JSON template is a data stream.
+#setup.template.json.data_stream: false
```

Comment:
There should be a changelog entry for this new option describing the problem it fixes.